HOSTING.md

```diff
@@ -0,0 +1,26 @@
+# Hosting NodeSchool YVR
+
+Are you interested in hosting NodeSchool Vancouver? We have a minimum set of requirements, and things other people have provided.
+
+## Minimum requirements
+
+* Seating and table space for 20+ people
+** Can be split into a couple rooms because we tend to split into smaller groups
+* Wifi for 20+ people
+* Power
+
+## Nice to have
+
+* Mentors
+** Usuaully a good idea to have people from your company/org interact with members of the community
+* Power strips
+** Some of the organizers have various power strips that can be loaned out if you don't have anything available
+* Job postings
+** Lots of node schoolers are looking for work, so its good to have someone or something that knows about your companies postings
+* Swag
+** Helps your company get remembered

```

Comment:
```suggestion
** Helps your company get remembered. (Stickers, print stickers ;) They're small, shareable, and a good conversation starter. Collapsible water bottles are great too.)
```

Reply:
I'm reluctant to tell people what swag they should get. They usually have marketing departments who figure out what works and what doesn't

Reply:
Fair point, thanks @halkeye :) I'm retracting the suggestion